src/include/sof/audio/codec_adapter/interfaces.h

```diff
@@ -11,9 +11,22 @@
 #ifndef __SOF_AUDIO_CODEC_INTERFACES__
 #define __SOF_AUDIO_CODEC_INTERFACES__
 
+#if CONFIG_CADENCE_CODEC
+#include <sof/audio/codec_adapter/codec/cadence.h>
+#endif
 /*****************************************************************************/
 /* Linked codecs interfaces						     */
 /*****************************************************************************/
-static struct codec_interface interfaces[0];
+static struct codec_interface interfaces[] = {
+	{
+		.id = 0xCADE01, /**< Cadence interface */
```

Comment:
Is this a version or an identifier for a certain Cadence component ? or does it apply to all Cadence ?

Reply:
@lgirdwood, it applies for codec which conforms to Cadence API